added ssl to config options (#56)
xapix-io/charts: xapix/README.md

When using AWS SNS to send emails, the kubernetes nodes need to be authorized to perform `ses:SendRawEmail`. You can attach this permission to the node instances when using [kops](https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#adding-additional-policies) for instance.

##### SSL Considerations

> If you're using ELB (Elastic Load Balancer) for SSL termination with Nginx as your ingress, you must ensure the `use-forwarded-headers` and `ssl-redirect` options are properly set in your ingress configuration. Documentation for those can be found ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-forwarded-headers) and ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-redirect).

Reviewer: ```suggestion
If you're using ELB (Elastic Load Balancer) for SSL termination with Nginx as your ingress, you must ensure that `use-forwarded-headers` is enabled and `ssl-redirect` is disabled in your ingress configuration. Documentation for those can be found ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-forwarded-headers) and ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-redirect).
```
